### PR TITLE
 I created the HTML for the login and sign up page, tested it as well

### PR DESCRIPTION
**Pull request by andreimastin**
I  created the html page for the sign up / login page, still rough but good.

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ class Meta: #all the variables you want to see
 
 @application.route('/', methods=['GET'])
 def default():
-    return {'Does this work?':'YES!'}
+    return {'Does this work?':'Mozys'}
```
Comment (JaadVdW) [inline application.py:144]: We sure on no return statement here?

File: application.py [hunk 2]
```diff
@@ -160,5 +160,5 @@ def check_login_creds():
 
 # Run server
 if __name__ == '__main__':
-    application.run(host='0.0.0.0') 
-    # application.run(debug=True)
+    #application.run(host='0.0.0.0') 
+     application.run(debug=True)
```
Comment (JaadVdW) [inline application.py:164]: If we are merging this to main we should have line 164 commented out and have line 163 active for deployment

**Review by JaadVdW — CHANGES_REQUESTED** [review]
Changes required.

**Review by jishaanm — COMMENTED** [review]
Could you please update the sign in page to be added to index.html and create a new page for sign up.

**Comment by andreimastin** [comment]
Closing and will resubmit, big changes to come, structure and location will differ

**Pull request closed without merging**